docs/tutorial.md

```diff
@@ -11,13 +11,24 @@ In this tutorial we use [HTTPie](https://httpie.org/) for HTTP requests, but of
 To get started, run our Docker image using this command:
 
 ```bash
-$ docker run -p 4000:4000 -p 4010:4010 -e HTTPS_CERTFILE=cert/selfsigned.pem -e HTTPS_KEYFILE=cert/selfsigned_key.pem accenture/reactive-interaction-gateway:2.0.2
+$ docker run \
+-e HTTPS_CERTFILE=cert/selfsigned.pem \
```

Comment:
I would omit also these 2 guys since we have note about SSL below.

Reply:
agree. Done